### PR TITLE
GROW-79 Replace home page hero with MG sign-up form

### PR DESCRIPTION
**Pull request by mlibazisi**

### CODE REVIEW

File: src/pages/LandingPages/MGCovid19/MGCovidHero.vue [hunk 1]
```diff
@@ -84,6 +84,10 @@ export default {
 		},
 		result({ data }) {
 			this.covidLandingActive = _get(data, 'general.mg_covid_active.value') === 'true' || false;
+			// Force the covid landing form to show when running the GROW-79 experiment
+			if (!this.covidLandingActive && this.isExperimentActive) {
```
Comment (mcstover) [inline src/pages/LandingPages/MGCovid19/MGCovidHero.vue:88]: We should not be overriding this feature flag.

It's purpose is two-fold; first as a gate in production today which you don't have to worry about and second as a kill switch for preventing entry into MG using the Covid Category and/or using the onetime deposit option.

Should there be a need to turn this off quickly we want to have a way that isn't subject to stale cookies in the browser as an experiment might be. That said, we would most likely want to turn off the experiment too.

Comment (mlibazisi) [reply src/pages/LandingPages/MGCovid19/MGCovidHero.vue]: Got it. Thanks!

File: src/pages/LandingPages/MGCovid19/MGCovidHero.vue [hunk 2]
```diff
@@ -95,6 +95,9 @@ export default {
 		category: {
 			type: String,
 			default: 'covid-relief'
+		},
+		isExperimentActive: {
```
Comment (emuvente) [inline src/pages/LandingPages/MGCovid19/MGCovidHero.vue:99]: It doesn't look like there are any usages of this currently. What is this going to be used for? Is that code still in progress?

Comment (mlibazisi) [reply src/pages/LandingPages/MGCovid19/MGCovidHero.vue:99]: @emuvente good catch. will remove it. thanks

**Pull request merged**